packages/action-password-update/src/components/PasswordUpdateDialog/PasswordInput.js

```diff
@@ -10,6 +10,7 @@ const PasswordInput = props => (
       value={props.value}
       onChange={e => props.onChange(e.target.value)}
       readOnly={props.readOnly === true}
+      disabled={props.readOnly === true}
```

Comment:
In this case we should set the input to `disabled` and remove the `readOnly` attribute. It doesn't make sense to set both of them (or does it?)